miden-crypto/src/merkle/mmr/tests.rs

```diff
@@ -1,3 +1,4 @@
+#![cfg(feature = "std")]
```

Comment:
Same comment about making the tests that require `Vec` conditional on `std`

Reply:
Same answer, it's about `rand::rng()` not `Vec`.